erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -181,6 +182,22 @@ bool WebRtcConnection::createOfferSync(bool video_enabled, bool audio_enabled, b
     }
   });
 
+  forEachTransceiver([this] (const std::shared_ptr<Transceiver> &transceiver) {
+    StreamDirection direction = StreamDirection::INACTIVE;
+    std::string stream_id("");
+    std::string transceiver_id(transceiver->getId());
+    if (!transceiver->isInactive()) {
+      if (transceiver->isSending()) {
+        direction = StreamDirection::SENDONLY;
+      } else {
+        direction = StreamDirection::RECVONLY;
+      }
+      stream_id = transceiver->getMediaStream()->getId();
+    }
```

Comment:
Shouldn't we be setting the direction to `StreamDirection::INACTIVE`?  I know we probably don't contemplate that case but nothing prevents us from filling in the data structure correctly.

Reply:
yes, StreamDirection::INACTIVE the default value, I set it on line 186.